fiasko_bro/validators.py

```diff
@@ -475,3 +475,11 @@ def is_nesting_too_deep(solution_repo, tab_size, max_indentation_level, whitelis
                     file_name = url_helpers.get_filename_from_path(file_path)
                     return 'too_nested', '{}:{}'.format(file_name, line_number)
             previous_line_indent = indentation_spaces_amount
+
+
+def has_no_string_sums(solution_repo, *args, **kwargs):
+    for tree in solution_repo.get_ast_trees():
+        for node in ast.walk(tree):
+            if (isinstance(node, ast.BinOp) and isinstance(node.left, ast.Str)
+            and isinstance(node.right, ast.Str)):
+                return 'has_string_sum', ''
```

Comment:
In the error message the validator should return the name of the file where the string summation takes place. The  argument `with_filenames` to `get_ast_trees` method will help you with that.